scripts/update-apk.sh

```diff
@@ -43,17 +43,17 @@ if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
 		if [[ $file = "open-event"* ]]; then
 			continue
 		fi
-		cp $file open-event-dev-${file%%}
+		mv $file open-event-dev-${file%%}
 	done
 fi
 
 # Signing Apps
 
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
     echo "Push to master branch detected, signing the app..."
-    cp app-release-unsigned.apk app-release-unaligned.apk
-	jarsigner -verbose -tsa http://timestamp.comodoca.com/rfc3161 -sigalg SHA1withRSA -digestalg SHA1 -keystore ../scripts/key.jks -storepass $STORE_PASS -keypass $KEY_PASS app-release-unaligned.apk $ALIAS
-	${ANDROID_HOME}/build-tools/27.0.3/zipalign -v -p 4 app-release-unaligned.apk app-release.apk
+    cp open-event-master-app-playStore-release-unsigned.apk app-playStore-release-unaligned.apk
```

Comment:
`app-playStore-release-unaligned.apk` Wrong format

Reply:
should i rename it to  `open-event-master-app-playStore-release-unaligned.apk` ?

Reply:
Yes

Reply:
updated